custom-elements/reactions/META.yml

```diff
@@ -0,0 +1,9 @@
+links:
+- product: safari
+  url: https://github.com/web-platform-tests/wpt/issues/21854
```

Comment:
As per the note I just put in that GH issue, I think this should be tracked via the webkit bug. Which is resolved WONTFIX, but that is ok - being able to track that a browser has said they wont fix something is still useful triage info!

Reply:
I was doubting if putting this link or using the WebKit one https://bugs.webkit.org/show_bug.cgi?id=182671. I thought the github/WPT one gave more context about the failures in WPT tests. Do you think I should change to the WebKit one?

Reply:
Here I think we should use the WebKit one, because there is no WPT bug - as in, this isn't failing due to an infrastructure issue or an issue with the tests themselves, it is a browser failure.

Reply:
Ok! Will change it